crates/codegen/grammar/src/grammar.rs

```diff
@@ -0,0 +1,102 @@
+use std::collections::{BTreeSet, HashMap};
+
+use semver::Version;
+
+use super::*;
+
+pub struct Grammar {
+    pub name: String,
+    pub versions: BTreeSet<Version>,
+    pub elements: HashMap<&'static str, GrammarElement>,
+    // pub default_parser: ParserDefinitionRef,
```

Comment:
Should we remove the commented out code?

Reply:
Done